dnf/base.py

```diff
@@ -1311,7 +1311,7 @@ def query_for_repo(query):
         q = self.sack.query()
         if pattern is not None:
             subj = dnf.subject.Subject(pattern, ignore_case=ic)
-            q = subj.get_best_query(self.sack, with_provides=False)
+            q = subj.get_best_query(self.sack)
```

Comment:
Not sure whatever this is a good idea. The same question is valid for repoquery command.

Reply:
I'll drop these commits and fix it in the doc.